cocotb/share/makefiles/Makefile.pylib.Msys

```diff
@@ -44,4 +44,4 @@ export PATH := $(LIB_DIR):$(PATH)
 export PYTHONHOME := $(shell $(PYTHON_BIN) -c "from distutils.sysconfig import get_config_var; print(get_config_var('prefix'))")
 
 # For tcl scripts the path to dll has to be in form of C:/User/..
-export LIB_DIR := $(shell sh -c 'cd $(LIB_DIR) && pwd -W')
+export LIB_DIR := $(shell cygpath -m "$(LIB_DIR)")
```

Comment:
Perhaps this should be single quotes, I don't know enough about makefiles.

Reply:
Line 44 (above) uses `"` so should be safe.